.github/workflows/docker.yml

```diff
@@ -0,0 +1,159 @@
+name: Docker unstable release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  ssr:
+    name: Build SSR images
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Get commit hash
+        run: echo "SHORT_SHA=${GITHUB_SHA::8}" >> $GITHUB_ENV
```

Comment:
just noting this here again in case you want to unify the steps (was unsure based on your [last comment](https://github.com/jellyfin/jellyfin-vue/pull/767#discussion_r582300633) whether you wanted to unify them)